web-api/smoke-tests.ts

```diff
@@ -1,11 +1,11 @@
 import { CognitoIdentityProvider } from '@aws-sdk/client-cognito-identity-provider';
-import AWS from 'aws-sdk';
+import { getServices } from './importHelpers';
 import assert from 'assert';
 import axios from 'axios';
 
 const ENV = process.argv[2];
 const REGION = process.argv[3];
-const DEPLOYING_COLOR = process.argv[4];
+// const DEPLOYING_COLOR = process.argv[4];
```

Comment:
Looks like this was borrowed from a bash script once upon a time. And this variable was used on line 13. Maybe line 13 can be fixed to use backticks to be a template string. Then again, it doesn't appear that `DEPLOYING_COLOR` is actually required.

Reply:
Good catch! I think we actually want to keep this: it allows us to run these tests against any color by passing a color in.

The most recent commit restores this functionality.